libs/milvus/pyproject.toml

```diff
@@ -4,7 +4,7 @@ build-backend = "poetry.core.masonry.api"
 
 [tool.poetry]
 name = "langchain-milvus"
-version = "0.1.6rc2"
+version = "0.1.7"
```

Comment:
you can keep this line unchanged : `version = "0.1.6rc2"` . we will change the version number when we are about to release the next version. btw actually next version is 0.1.6